tests/saitests/sai_qos_tests.py

```diff
@@ -781,6 +785,14 @@ def runTest(self):
             for cntr in egress_counters:
                 assert(xmit_counters[cntr] == xmit_counters_base[cntr])
 
+            # According to SONiC configuration lossless dscps are classified as follows:
+            # dscp  3 -> pg 3
+            # dscp  4 -> pg 4
+            pg_dropped_cntrs = sai_thrift_read_pg_drop_counters(self.client, port_list[src_port_id])
```

Comment:
Please check the sonic version and execute this block. These tests also run against 201811 and these counters are not supported there.  there is a similar check in get_counter_names()

Reply:
Done